src/defrag.c

```diff
@@ -1054,12 +1036,12 @@ void activeDefragCycle(void) {
         /* This array of structures holds the parameters for all defragmentation stages. */
         typedef struct defragStage {
             kvstore *kvs;
-            dictScanFunction *scanfn;
+            hashsetScanFunction scanfn;
             void *privdata;
         } defragStage;
         defragStage defrag_stages[] = {
             {db->keys, defragScanCallback, db},
-            {db->expires, scanCallbackCountScanned, NULL},
+            //{db->expires, scanCallbackCountScanned, NULL},
```

Comment:
?

Reply:
DELETEME

Reply:
When `db->keys` is defragged using `defragKey`, the pointer to the robj in expire is updated too. There's no reason to scan the defrag table. We defrag the hashset allocations though, using `kvstoreHashsetDefragInternals`.

Reply:
I'm deleting this commented out line. Not committed yet though, but I'll push it later.